app/index.php

```diff
@@ -1,19 +1,28 @@
 <?php
-session_start();
 require "../vendor/autoload.php";
+session_start();
 $conn = new \App\DbConnector();
 $store = new \App\Store($conn->getDb());
 
-if (!empty($_GET) && $_GET['min'] && $_GET['max']) {
-    $books = $store->getBooksWithinRange($_GET['min'], $_GET['max']);
-    $bookPrices = $store->getAllBookPrices();
-    $sortedBooks = new \App\SortBooks($bookPrices);
+if (isset($_GET['search'])) {
+    $searchTerm = $_GET["search"];
+    $searchGetTerm = 'search=' . $_GET['search'] . "&";
+    $searchTitle = new \App\SearchTitle($conn->getDb());
+    $books = $searchTitle->performSearch($searchTerm);
+    $sortBooks = new \App\SortBooks($books);
 } else {
+    $searchTerm = "";
+    $searchGetTerm = "";
     $books = $store->getAllBooks();
-    $sortedBooks = new \App\SortBooks($books);
+    $filteredBooksPrices = $store->getAllBookPrices();
+    $sortBooks = new \App\SortBooks($filteredBooksPrices);
 }
 
-$bookPrices = $sortedBooks->getBooksPriceAscending();
+if (!empty($_GET) && $_GET['min'] && $_GET['max']) {
```

Comment:
`&& $_GET['min']` is checking that min is non zero. What is this for? Is this meant to be checking `isset($_GET['min']`?